ospfd/ospf_ri.c

```diff
@@ -700,17 +686,17 @@ static int ospf_router_info_lsa_originate(void *arg)
 
 	int rc = -1;
 
-	if (OspfRI.status == disabled) {
+	if (OspfRI.status == disable) {
 		zlog_info(
 			"ospf_router_info_lsa_originate: ROUTER INFORMATION is disabled now.");
 		rc = 0; /* This is not an error case. */
-		goto out;
+		return rc;
 	}
 
 	/* Check if Router Information LSA is already engaged */
-	if (OspfRI.flags & RIFLG_LSA_ENGAGED) {
-		if (OspfRI.flags & RIFLG_LSA_FORCED_REFRESH) {
-			OspfRI.flags &= ~RIFLG_LSA_FORCED_REFRESH;
+	if CHECK_FLAG(OspfRI.flags, RIFLG_LSA_ENGAGED) {
```

Comment:
this looks really, really weird style-wise due to the missing ( between "if" and "CHECK_FLAG".

can you fix this to `if (CHECK_FLAG(...))`, i.e. add ()?

(it's like this in a few more locations)